mmv1/products/networkservices/Gateway.yaml

```diff
@@ -61,6 +61,48 @@ examples:
     primary_resource_id: 'default'
     vars:
       resource_name: 'my-gateway'
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_gateway_secure_web_proxy'
+    primary_resource_id: 'default'
+    vars:
+      certificate_name: 'my-certificate'
+      network_name: 'my-network'
+      subnetwork_name: 'my-subnetwork-name'
+      proxy_only_subnetwork_name: 'my-proxy-only-subnetwork'
+      policy_name: 'my-policy-name'
+      policy_rule_name: 'my-policyrule-name'
+      gateway_name_1: 'my-gateway1'
+      scope_1: 'my-default-scope1'
+    ignore_read_extra:
+      - "delete_swg_autogen_router_on_destroy"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: 'network_services_gateway_multiple_swp_same_network'
+    primary_resource_id: 'default'
+    vars:
+      certificate_name: 'my-certificate'
+      network_name: 'my-network'
+      subnetwork_name: 'my-subnetwork-name'
+      proxy_only_subnetwork_name: 'my-proxy-only-subnetwork'
+      policy_name: 'my-policy-name'
+      policy_rule_name: 'my-policyrule-name'
+      gateway_name_1: 'my-gateway1'
+      gateway_name_2: 'my-gateway2'
+      scope_1: 'my-default-scope1'
+      scope_2: 'my-default-scope2'
+    ignore_read_extra:
+      - "delete_swg_autogen_router_on_destroy"
+virtual_fields:
+  - !ruby/object:Api::Type::Boolean
+    name: 'delete_swg_autogen_router_on_destroy'
```

Comment:
Have you by chance tested this field if it works as intended when setting as `false`?

Reply:
Yes. I generated the beta-provider and played with that. If it sets to false, it will leave the swg-autogen-router behind.